### PR TITLE
Remove more explicit search response instances in tests

### PR DESCRIPTION
**Pull request by original-brownbear**
Another round of fixing these using the new utils added.

### CODE REVIEW

**Review by iverase — APPROVED** [review]
LGTM

File: server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/MissingValueIT.java [hunk 1]
```diff
@@ -54,161 +54,186 @@ protected void setupSuiteScopeCluster() throws Exception {
...
+            Terms terms = response.getAggregations().get("my_terms");
+            assertEquals(2, terms.getBuckets().size());
+            assertEquals(1, terms.getBucketByKey("3").getDocCount());
+            assertEquals(1, terms.getBucketByKey("4").getDocCount());
+        });
+        assertNoFailuresAndResponse(prepareSearch("idx").addAggregation(terms("my_terms").field("long").missing(3)), response -> {
+            assertNoFailures(response);
+            Terms terms2 = response.getAggregations().get("my_terms");
+            assertEquals(1, terms2.getBuckets().size());
+            assertEquals(2, terms2.getBucketByKey("3").getDocCount());
```
Comment (DaveCTurner) [inline server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/MissingValueIT.java:98]: IntelliJ being silly with its variable names I think

```suggestion
            Terms terms = response.getAggregations().get("my_terms");
            assertEquals(1, terms.getBuckets().size());
            assertEquals(2, terms.getBucketByKey("3").getDocCount());
```

**Comment by original-brownbear** [comment]
Thanks @iverase !

**Review by DaveCTurner — APPROVED** [review]
LGTM (one nit)

**Comment by DaveCTurner** [comment]
too slow 🐌

File: server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/MissingValueIT.java (hunk 1 above)
Comment (original-brownbear) [reply server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/MissingValueIT.java:98]: I admit I did this by hand :/ no clue why ... sorry!

**Pull request merged**